Lib/cmd.py

```diff
@@ -108,7 +108,7 @@ def cmdloop(self, intro=None):
                 import readline
                 self.old_completer = readline.get_completer()
                 readline.set_completer(self.complete)
-                if 'libedit' in getattr(readline, '__doc__', ''):
+                if readline.backend == "editline":
```

Comment:
Because of 
```
Lib/site.py:447:        if readline.backend == 'editline':
Lib/test/test_pdb.py:3296:        if readline.backend == "editline":
Lib/test/test_readline.py:149:        self.assertIn(readline.backend, ("readline", "editline"))
Lib/test/test_readline.py:177:            if readline.backend == "editline":
Lib/test/test_readline.py:204:is_editline = readline.backend == "editline"
Lib/test/test_readline.py:22:    is_editline = readline.backend == "editline"
```